mayastor/src/bdev/nexus/nexus_child_error_store.rs

```diff
@@ -48,6 +51,11 @@ pub struct NexusErrStore {
     records: Vec<NexusChildErrorRecord>,
 }
 
+pub enum NexusErrStoreQuery {
```

Comment:
as all the code is already namespaced (i.e bdev::nexus::nexus_child_error_store) there is no need to make that:

bdev::nexus::nexus_child_error_store::NexusErrStoreQuery -- that is all very, very verbose, would you agree? Ideally it would not have been named nexus_child_error_store either but thats not part of this change so thats OK.

Reply:
I tend to prefer verbose names, but if the house style is to keep them short, I'll do so.